addons/mass_mailing/models/mail_mail.py

```diff
@@ -82,7 +82,7 @@ def _prepare_outgoing_list(self, recipients_follower_status=None):
 
             # replace links in body
             if not tools.is_html_empty(email_values['body']):
-                if f'{base_url}/unsubscribe_from_list' in email_values['body']:
+                if f'{base_url}/unsubscribe_from_list' in email_values['body'] and not self.env.context.get('send_as_test'):
```

Comment:
I would instead redirect the unsubscribe url to ``mailing/my`` which is a generic page for managing subscriptions. That way we have a working link that does not perform any real unsubscribe.

Reply:
This is what the route for `/unsubscribe_from_list` [does](https://github.com/odoo-dev/odoo/blob/a23d72a22dfce855a112bd9143044e50ccef7890/addons/mass_mailing/controllers/main.py#L320-L324). So by blocking this replacement we have the smallest diff possible for the behavior. But if you'd prefer that I explicitly modify the link so that it contains `/mailing/my` I can do it.

Reply:
Oh we did that ? Noice :D